msteams-platform/whats-new.md

```diff
@@ -189,6 +189,7 @@ Developer preview is a public program that provides early access to unreleased T
 
 | **Date** | **Notes** | **Changed topics** |
 | -------- | --------- | ------------------ |
+|06/17/2021|Uninstall event| [Uninstall event in conversation events](bots/how-to/conversations/subscribe-to-conversation-events.md#uninstall-event)|
```

Comment:
a more accurate description would probably be: Uninstall behavior for personal app with bot| [Uninstall behavior updates in personal apps with bots](bots/how-to/conversations/subscribe-to-conversation-events.md#uninstall-event)|